MANIFEST.in

```diff
@@ -1,4 +0,0 @@
-include README.md
-include LICENSE
-include requirements.txt
-include cloudpathlib/py.typed
```

Comment:
Do we need to explicitly include `py.typed` somehwere?

Reply:
Nope, I had checked both the wheel and sdist and it was already in there with default behavior.